source/lectures/oop/properties.md

```diff
@@ -136,7 +136,7 @@ tags:
     ```
     public int Width { get; set;  }
     ```
-    This is called *automatic properties* (or *auto-properties*).
+    This is called *automatic properties*.
```

Comment:
I think having the term listed at least one has some merit, actually. I would recommend leaving it there.